.github/workflows/validate.yml

```diff
@@ -23,6 +23,16 @@ jobs:
           # must be specified without patch version
           version: v1.40
 
+  codespell:
```

Comment:
Curious; have you also tried enabling `misspell` in golang-ci-lint? Mostly wondering if it would catch the same errors (it would save us from having to run a separate linter for this); https://golangci-lint.run/usage/linters

Reply:
Oh, I guess `misspell` may only be looking for `.go` files 🤔

Reply:
That's right; as part of golangci-lint it will _probably_ only check `.go` files.

It is also
- apparently unmaintained since March 2018 (judging by its git repo at https://github.com/client9/misspell)
- can only find two misspellings in the very same codebase as this PR, when run as a standalone binary (checks all files);

```console
[kir@kir-rhat ~]$ go1.17 install github.com/client9/misspell/cmd/misspell@latest

[kir@kir-rhat runc]$ misspell * | grep -v ^vendor/
libcontainer/cgroups/ebpf/ebpf_linux.go:137:23: "succeded" is a misspelling of "succeeded"
libcontainer/cgroups/systemd/systemd_test.go:344:22: "contianer" is a misspelling of "container"
```

(the last typo it found is the one codespell can't see, but I've added the fix to this PR)

Reply:
In addition to the above, there's something that makes me think you'd favor codespell over it 🙃 

Once I've rebased:
```
[kir@kir-rhat runc]$ misspell * | grep -v ^vendor/
MAINTAINERS:8:0: "Sebastiaan" is a misspelling of "Sebastian"
```

Reply:
LOL!  😂